src/server.js

```diff
@@ -206,7 +218,8 @@ const pe = new PrettyError();
 pe.skipNodeFiles();
 pe.skipPackage('express');
 
-app.use((err, req, res, next) => { // eslint-disable-line no-unused-vars
+app.use((err, req, res, next) => {
+  // eslint-disable-line no-unused-vars
   console.log(pe.render(err)); // eslint-disable-line no-console
```

Comment:
I don't know. I didn't really change this file a lot. These changes are just prettier formatting it.